src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const hasPartner =
+    person.sex === 'f' ? (
+      <p className="Person__partner">{person.partnerName} is my husband</p>
+    ) : (
+      <p className="Person__partner">{person.partnerName} is my wife</p>
+    );
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {person.name}</h2>
+      {person.age ? <p className="Person__age">I am {person.age}</p> : null}
+      {person.isMarried === true ? (
+        hasPartner
```

Comment:
here just {person.partnerName} is my {partner}

Reply:
Added  {person.partnerName} is my {partner}.